internal/query/session.go

```diff
@@ -283,6 +307,8 @@ func prepareSessionQuery(ctx context.Context, db prepareDatabase) (sq.SelectBuil
 				otpEmailCheckedAt   sql.NullTime
 				metadata            database.Map[[]byte]
 				token               sql.NullString
+				userAgentIP         sql.NullString
+				userAgentHeader     []byte
```

Comment:
`database.Map[[]byte]`?

Reply:
Yes, you are right. overlooked that. Using Map now.